test/route.6.test.js

```diff
@@ -63,38 +62,41 @@ test('Will not create a HEAD route that is not GET', t => {
     }
   })
 
-  fastify.inject({
-    method: 'HEAD',
-    url: '/more-coffee'
-  }, (error, res) => {
-    t.error(error)
-    t.equal(res.statusCode, 200)
-    t.equal(res.headers['content-type'], 'application/json; charset=utf-8')
-    t.same(res.body, '')
-  })
+  {
```

Comment:
Why are these being wrapped in blocks?

Reply:
Hi, it is done to avoid the res variables from conflicting. 
The same kind of pattern can be already found in `500.test.js` on line 91 for example.

Do you have any proposal or preference over this?

Reply:
I would do the following:

```js
let res = await aRequest()
assert.somethingAbout(res)
/* more code */

res = await aSecondRequest()
assert.somethingAbout(res)
```

I do think blocks can be useful, but I'm not convinced we really need them to structure these tests.

Reply:
make sense, changed with no blocks!